Documentation/Configuration/Typo3ConfVars/SYS.rst

```diff
@@ -658,6 +658,33 @@ configurations.
         If enabled, the :ref:`Content Security Policy <content-security-policy>`
         is applied in frontend scope.
 
+    ..  _typo3ConfVars_sys_features_security.frontend.allowInsecureFrameOptionInShowImageController:
+
+    ..  confval:: security.frontend.allowInsecureFrameOptionInShowImageController
+        :name: globals-typo3-conf-vars-sys-features-security-frontend-aallowInsecureFrameOptionInShowImageController
+        :Path: $GLOBALS['TYPO3_CONF_VARS']['SYS']['features']['security.frontend.allowInsecureFrameOptionInShowImageController']
+        :type: bool
+        :Default: false
+
+        ..  versionadded:: 13.1, 12.4.15, 11.5.37
+
+        The show image controller (eID `tx_cms_showpic`) lacks a cryptographic
```

Comment:
Don't have the time now but I think we should rephrase that less changelogy and much shorter:

To restore behavior of older TYPO3 versions, this parameter can be enabled to allow non-cryptogrpahically signed URLs (lacking 'cHash'). It is strongly suggested to leave it turned off (default).

Maybe a "changelog entry xxx reveals reasoning for this" reference?

Reply:
Why dont we merge it like this for now and you improve the text in a follow up once you have time? At least we have the option mentioned then

Reply:
I now made a proposal to change it, is that ok for you @linawolf ?